### PR TITLE
Initializing chunking with empty 2d array

### PR DESCRIPTION
**Pull request by mboudreau**

### CODE REVIEW

**Review by KAbakumovAtl — APPROVED** [review]

File: src/jira/client/index.ts [hunk 1]
```diff
@@ -281,20 +281,27 @@ const batchedBulkUpdate = async (
 	options?: { preventTransitions: boolean }
 ) => {
 	// Initialize with an empty chunk of commits so we still process the request if there are no commits in the payload
-	const commitChunks: JiraCommit[][] = _.chunk(dedupCommits(data.commits), 400);
+	let commitChunks: JiraCommit[][] = _.chunk(dedupCommits(data.commits), 400);
```
Comment (ccrolf) [inline src/jira/client/index.ts:284]: Let's add a feature flag, so there's no functional change when the code goes to production

**Review by ccrolf — CHANGES_REQUESTED** [review]

File: src/jira/client/index.ts (hunk 1 above)
Comment (mboudreau) [reply src/jira/client/index.ts:284]: :+1:

File: src/jira/client/index.ts [hunk 2]
```diff
@@ -280,11 +281,45 @@ const batchedBulkUpdate = async (
 	logger?: Logger,
 	options?: { preventTransitions: boolean }
 ) => {
-	// Initialize with an empty chunk of commits so we still process the request if there are no commits in the payload
-	const commitChunks: JiraCommit[][] = _.chunk(dedupCommits(data.commits), 400);
+	if (booleanFlag(BooleanFlags.REMOVE_WHILE_LOOP, true)) {
```
Comment (ccrolf) [inline src/jira/client/index.ts:284]: Should the default value be false? Otherwise, this code would roll out if LaunchDarkly has an outage

Comment (mboudreau) [reply src/jira/client/index.ts:284]: but then it won't be enabled by default on your local...

**Review by rachellerathbone — APPROVED** [review]

**Review by KAbakumovAtl — APPROVED** [review]

**Review by ccrolf — APPROVED** [review]

**Pull request merged**